main.py

```diff
@@ -1,16 +1,17 @@
-from question_model import Question
-from data import question_data
 from quiz_brain import QuizBrain
 from ui import QuizInterface
+from questiondb import QuestionBank
 
-question_bank = []
-for question in question_data:
-    question_text = question["question"]
-    question_answer = question["correct_answer"]
-    new_question = Question(question_text, question_answer)
-    question_bank.append(new_question)
 
-
-quiz = QuizBrain(question_bank)
+question_bank = QuestionBank()
+quiz = QuizBrain(question_bank.questions)
 quiz_ui = QuizInterface(quiz)
```

Comment:
caly czas UI jest modulem zarzadzajacym, Brain jest parametrem UI, UI kontroluje ktore funkcje brain są odpalane

Reply:
OK, to zmienię jak będę miał więcej czasu - ciekawy problem swoją drogą

Reply:
ok